src/D2L.Bmx/OktaAuthenticator.cs

```diff
@@ -12,7 +12,8 @@ BmxConfig config
 	public async Task<IOktaApi> AuthenticateAsync(
 		string? org,
 		string? user,
-		bool nonInteractive
+		bool nonInteractive,
+		bool ignoreCache = false
```

Comment:
there's a default here, but you also explicitly specified the default value in print & write handler.
Either remove this default value, or actually utilize it in the other two handlers.

Reply:
Ah yea. I've removed the default value since none of the other parameters use it